CHANGELOG.md

```diff
@@ -5,13 +5,85 @@ Releases are named after famous [Cricket](https://en.wikipedia.org/wiki/Cricket)
 
 
 
-## [Unreleased](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/compare/0.7.0..master)
+## [Unreleased](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/compare/0.8.0..master)
 
 ### Breaking changes
+- Github organisation OAuth: `auth.github.org_whitelist` has been renamed to `auth.github.orgWhitelist`
 
 ### New Features
 
 
+## [0.8.0](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/compare/0.7.0..0.8.0) - [Richie Benaud](https://en.wikipedia.org/wiki/Richie_Benaud) - 2019-01-24
+
+This release contains JupyterHub version 0.9.4. It requires Kubernetes >= 1.11 and Helm >= 2.11.0.
+See [the Helm Chart repository](https://github.com/jupyterhub/helm-chart#versions-coupled-to-each-chart-release) for
+a list of relevant dependencies for all Helm Chart versions.
+
+It contains new features, additional configuration options, and bug fixes.
+
+### Upgrading from 0.7
+
+TODO: Instructions
```

Comment:
I think using `--force` should be recommended at all times, I don't remember if it is required but I think it may be.